infra/build/functions/request_build.py

```diff
@@ -23,17 +23,37 @@
 
 import build_lib
 import build_project
+from datastore_entities import BuildsHistory
 from datastore_entities import Project
 
 BASE_PROJECT = 'oss-fuzz-base'
+MAX_BUILD_HISTORY_LENGTH = 64
+
+
+def update_build_history(project_name, build_id, build_tag_suffix):
+  """Update build history of project."""
+  project_key = ndb.Key(BuildsHistory, project_name + build_tag_suffix)
+  project = project_key.get()
+
+  if project is None:
+    project = BuildsHistory(id=project_name + '-' + build_tag_suffix,
+                            build_tag_suffix=build_tag_suffix,
+                            project=project_name,
+                            build_ids=[])
+
+  if len(project.build_ids) >= MAX_BUILD_HISTORY_LENGTH:
+    project.build_ids.pop(0)
+
+  project.build_ids.append(build_id)
+  project.put()
 
 
 def get_project_data(project_name):
   """Retrieve project metadata from datastore."""
   with ndb.Client().context():
     query = Project.query(Project.name == project_name)
     project = query.get()
-    if project is None:
+    if not Project:
```

Comment:
Project shouldn't be capitalised. Otherwise this check isn't working as expected. Please add a test here as well so this is caught.